### PR TITLE
chore(docs): document kafka connector backoff configuration options

### PR DESCRIPTION
**Pull request by jerrinot**

### CODE REVIEW

File: docs/third-party-tools/kafka/questdb-kafka.md [hunk 1]
```diff
@@ -191,6 +191,8 @@ The connector supports the following configuration options:
 | username                          | `string`  | user1                                                       | admin              | User name for QuestDB. Used only when token is non-empty   |
 | token                             | `string`  | <sub>QgHCOyq35D5HocCMrUGJinEsjEscJlC</sub>                  | N/A                | Token for QuestDB authentication                           |
 | tls                               | `boolean` | true                                                        | false              | Use TLS for QuestDB connection                             |
+| retry.backoff.ms                  | LONG      | 1000                                                        | 3000               | Connection retry interval in milliseconds                  |
```
Comment (amyshwang) [inline docs/third-party-tools/kafka/questdb-kafka.md:194]:
```suggestion
| retry.backoff.ms                  | `long`      | 1000                                                        | 3000               | Connection retry interval in milliseconds                  |
```

File: docs/third-party-tools/kafka/questdb-kafka.md [hunk 2]
```diff
@@ -191,6 +191,8 @@ The connector supports the following configuration options:
...
+| max.retries                       | LONG      | 1                                                           | 10                 | Maximum no. connection of retry attempts                   |
```
Comment (amyshwang) [inline docs/third-party-tools/kafka/questdb-kafka.md:195]: Is it the number of attempt to reconnect, right?
```suggestion
| max.retries                       | `long`      | 1                                                           | 10                 | Maximum number of connection retry attempts                   |
```

**Review by amyshwang — COMMENTED** [review]
Minor suggestions for formatting

File: docs/third-party-tools/kafka/questdb-kafka.md (hunk 2 above)
Comment (jerrinot) [reply docs/third-party-tools/kafka/questdb-kafka.md]: correct

**Review by amyshwang — APPROVED** [review]

**Pull request merged**